health-services/project-factory/src/server/utils/microplanUtils.ts

```
@@ -0,0 +1,383 @@
+  request.body.sheetErrorDetails = request?.body?.sheetErrorDetails ? [...request?.body?.sheetErrorDetails, ...overallInconsistencies] : overallInconsistencies;
+}
+
+function validateNationalDuplicacy(request: any, userMapping: any, phoneNumberKey: any) {
+  const duplicates: any[] = [];
+
+  for (const phoneNumber in userMapping) {
+    const roleMap: any = {};
+    const users = userMapping[phoneNumber];
+
+    for (const user of users) {
+      if (user.role && user.role.startsWith("Root ")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)